setup.cfg

```diff
@@ -0,0 +1,27 @@
+[metadata]
+name = git-filter-repo
+url = https://github.com/newren/git-filter-repo
+project_urls = 
+    Source = https://github.com/newren/git-filter-repo
+    Issues = https://github.com/newren/git-filter-repo/issues/
+description = git filter-branch replacement
+version = 2.25.0
```

Comment:
I don't like hardcoding the version in one (or more) of the files in the repo; can we auto-generate this instead based on the current tag?

Reply:
Yep, can do that.